tasks/20230526-gauge-working-balance-helper/test/L1GaugeHelper.fork.ts

```diff
@@ -0,0 +1,210 @@
+import hre from 'hardhat';
+import { expect } from 'chai';
+import { Contract } from 'ethers';
+import { getForkedNetwork, Task, TaskMode, describeForkTest, getSigners, impersonate, instanceAt } from '@src';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { fp } from '@helpers/numbers';
+import { MAX_UINT256, ZERO_ADDRESS } from '@helpers/constants';
+import { WeightedPoolEncoder } from '@helpers/models/pools/weighted/encoder';
+import { MONTH, currentTimestamp, advanceTime } from '@helpers/time';
+
+describeForkTest('GaugeWorkingBalanceHelper-L1', 'mainnet', 17258776, function () {
```

Comment:
Does the test break if we select a newer block #?

(same for the remaining fork tests)

Reply:
I could advance the L1's a bit, but the L2 is very sensitive